data/anaconda-gtk.css

```diff
@@ -84,7 +84,7 @@ levelbar.discrete trough block.filled.high {
 
 /* theme colors/images */
 
-@define-color theme_bg_color @fedora;
+@define-color fedora_bg_color @fedora;
```

Comment:
The color can be set to a different value in different products. Maybe it could be called `product_bg_color` then?